perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -101,6 +101,14 @@ sub geocode_postcode {
         return $parks_lookup;
     }
 
+    # split postcode with Lewisham
+    if ($s =~ /BR1\s+4EY\s+$/i) {
```

Comment:
This matches at a minimum "BR1 4EY " or with more spaces. I assume you meant `*`. There is an existing example in Bexley.pm.